Formula/s/slither-analyzer.rb

```diff
@@ -233,15 +239,11 @@ class SlitherAnalyzer < Formula
     sha256 "566db86717cf8080b99b58b083b773a908ae40f06681e87e589a976faf8246bf"
   end
 
-  # Drop setuptools dep
-  # https://github.com/crytic/slither/pull/2290
-  patch do
-    url "https://github.com/crytic/slither/commit/93cbf7d887a930e20c1ea8cf543ec2f41e84aadd.patch?full_index=1"
-    sha256 "6c6278b86c8d75fb01a50715a8891ab8a94ec6d4372092440f1cefd67fbe0940"
-  end
-
   def install
-    virtualenv_install_with_resources
+    venv = virtualenv_create(libexec, "python3.12")
+    resource("ckzg").stage { venv.pip_install "bindings/python" }
```

Comment:
I tried upgrading this locally; a change like this over this PR resolves the build failure

```diff
-    resource("ckzg").stage { venv.pip_install "bindings/python" }
+    resource("ckzg").stage do
+      inreplace "bindings/python/setup.py", 'include_dirs=["../../inc"', "include_dirs=[\"#{Formula["blst"].opt_include}/blst\""
+      venv.pip_install "bindings/python"
+    end
```